src/components/portrait/portrait.component.js

```diff
@@ -179,108 +179,71 @@ class Portrait extends React.Component {
     return context;
   }
 
-  /**
-   * Maps size to width/height value
-   *
-   * @method numericSizes
-   * @return {Object}
-   */
-  get numericSizes() {
-    return {
-      'extra-small': '25',
-      small: '30',
-      'medium-small': '40',
-      medium: '60',
-      'medium-large': '70',
-      large: '100',
-      'extra-large': '120'
-    };
-  }
-
-  /**
-   * Main Class getter
-   *
-   * @method mainClasses
-   * @return {String} Main className
-   */
-  get mainClasses() {
-    return classNames(
-      'carbon-portrait',
-      'carbon-portrait--image',
-      `carbon-portrait--${this.props.size}`,
-      `carbon-portrait--${this.props.shape}`,
-      this.props.className, {
-        'carbon-portrait--dark-background': this.props.darkBackground
-      }
-    );
-  }
-
-  /**
-   * Return the html for the initials image.
-   *
-   * @method initialsImage
-   * @return {Object}
-   */
-  get initialsImage() {
-    // if not using src, generate initials for potential fallback
-    if (this.props.src) { return null; }
-    if (!this.props.initials) { return this.sansInitialsImage; }
-
-    return (
-      <img
-        data-element='initials'
-        className='carbon-portrait__img carbon-portrait__initials'
-        src={ this.generateInitials }
-        alt={ this.props.alt }
-      />
-    );
-  }
-
-  /**
-   * Return the html for the avatar image.
-   *
-   * @method avatarImage
-   * @return {Object}
-   */
-  get avatarImage() {
-    return (
-      <img
-        data-element='user-image'
-        className='carbon-portrait__img carbon-portrait__avatar'
-        src={ this.imgSrc }
-        alt={ this.props.alt }
-      />
-    );
-  }
-
-  /**
-   *
-   * Return the html for the Icon displayed in the event that both the image and initials are missing
-   *
-   * @method sansInitialsImage
-   * @return {Object}
-   */
-  get sansInitialsImage() {
-    return (
-      <Icon
-        className='carbon-portrait__img carbon-portrait__sans-initials'
-        type='individual'
-      />
-    );
-  }
-
   /**
    * Renders the component.
    *
    * @method render
    * @return {Object} JSX
    */
   render() {
+    const {
+      alt, className, darkBackground, gravatar, initials, shape, size, src
+    } = this.props;
+
     return (
-      <div className={ this.mainClasses } { ...tagComponent('portrait', this.props) }>
-        { this.initialsImage }
-        { this.avatarImage }
-      </div>
+      <StyledPortrait
+        size={ size }
+        shape={ shape }
+        className={ className }
```

Comment:
Do we still need className when we have styled-components?

Reply:
@dawidzarzycki: It looks like the `Portrait` component has supported the `className` prop for at least a couple of years (see [here](https://github.com/Sage/carbon/blame/4dc5509/src/components/portrait/portrait.component.js#L24-L27) and [here](https://github.com/Sage/carbon/blame/4dc5509/src/components/portrait/__definition__.js#L31)). This allows a custom class to be applied to the component, eg:

```jsx
<Portrait className="my-custom-class" />
```

Does the component still need to support this functionality?

Reply:
I'm not sure. What I see in the different cases that devs just remove className in DLS components

Reply:
OK, I've removed `className` in 47c86d2 :thumbsup: